Buildings/Fluid/Air/AirHandlingUnit.mo

```diff
@@ -0,0 +1,258 @@
+        iconTransformation(extent={{-120,-10},{-100,10}})));
+  Modelica.Blocks.Interfaces.RealInput XSet_w(unit="kg/kg")
+    "Set point for water vapor mass fraction in kg/kg total air of the fluid that leaves port_b"
+    annotation (Placement(transformation(extent={{-140,-10},{-100,30}}),
+        iconTransformation(extent={{-120,10},{-100,30}})));
+  Modelica.Blocks.Sources.RealExpression dT(y(unit="K")=T_inflow_hea - TSet)
+    "Difference between inlet temperature and temperature setpoint of the reheater"
+    annotation (Placement(transformation(extent={{-62,-10},{-42,10}})));
+
+  Buildings.Fluid.Humidifiers.SteamHumidifier_X hum(
+    final allowFlowReversal=allowFlowReversal2,
+    final m_flow_small=m2_flow_small,
+    final show_T=show_T,
+    final massDynamics=massDynamics,
+    final tau=tauHum,
+    final homotopyInitialization=homotopyInitialization,
+    final dp_nominal=0,
+    final m_flow_nominal=m2_flow_nominal,
+    final mWatMax_flow=mWatMax_flow,
+    each final X_start=X_start,
+    final from_dp=from_dp2,
+    final linearizeFlowResistance=linearizeFlowResistance2,
+    final deltaM=deltaM2,
+    redeclare final package Medium = Medium2)
+    "Humidifier" annotation (Placement(
+        transformation(
+        extent={{-10,10},{10,-10}},
+        rotation=180,
+        origin={20,-60})));
+  Buildings.Fluid.Air.BaseClasses.ElectricHeater eleHea(
+    final allowFlowReversal=allowFlowReversal2,
+    final show_T=show_T,
+    final m_flow_small=m2_flow_small,
+    final energyDynamics=energyDynamics,
+    final tau=tauEleHea,
+    final homotopyInitialization=homotopyInitialization,
+    final dp_nominal=0,
+    final QMax_flow=QHeaMax_flow,
+    final eta=etaHea,
+    final m_flow_nominal=m2_flow_nominal,
+    final T_start=T_start,
+    final from_dp=from_dp2,
+    final linearizeFlowResistance=linearizeFlowResistance2,
+    final deltaM=deltaM2,
+    redeclare final package Medium = Medium2)
+    "Electric heater" annotation (
+      Placement(transformation(
+        extent={{10,-10},{-10,10}},
+        rotation=0,
+        origin={-22,-60})));
+
+  Buildings.Fluid.Air.BaseClasses.ReheatControl heaCon(
+    final pre_yVal_start=pre_yVal_start,
+    final pre_dT_start=pre_dT_start,
+    final yValLow=yValLow,
+    final yValHig=yValHig,
+    final dTLow=dTLow,
+    final dTHig=dTHig)
+    "Reheater on/off controller"
+    annotation (Placement(transformation(extent={{-10,-10},{10,10}},
+        rotation=0,
+        origin={-12,6})));
+
+protected
+  Medium2.Temperature T_inflow_hea= Medium2.temperature(state=Medium2.setState_phX(port_b2.p,
+                           noEvent(actualStream(port_b2.h_outflow)),
+                           noEvent(actualStream(port_b2.Xi_outflow))))
+      "Temperature of inflowing fluid at port_a of reheater";
+
+
+equation
+
+
+  connect(TSet, eleHea.TSet)
+  annotation (Line(points={{-120,-10},{-88,-10},{-88,-28},{-4,-28},{-4,-52},{
```

Comment:
re-organize the annotation text to make it more readable, e.g. "increase indent", avoid unnecessary line change, etc.